packages/typespec-vscode/src/tsp-language-client.ts

```diff
@@ -0,0 +1,124 @@
+import { ExtensionContext, LogOutputChannel, RelativePattern, workspace } from "vscode";
```

Comment:
Refactor the language client code to a class to a separated file and fix the restart bug (in restart method) in the meantime